pkg/space/space.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // NewSpace creates a space CR for a UserSignup object.
-func NewSpace(userSignup *toolchainv1alpha1.UserSignup, targetClusterName string, compliantUserName, tier string) *toolchainv1alpha1.Space {
+func NewSpace(userSignup *toolchainv1alpha1.UserSignup, targetClusterName string, compliantUserName, tier string, disableInheritance bool) *toolchainv1alpha1.Space {
```

Comment:
Do we actually need this? the root-space should always have `DisableInheritance=false` - there is no point in disabling the inheritance there. Also, when I look at the places where you call this function, it always sets `false`, or am I missing something?

Reply:
@MatousJobanek I've addressed this in https://github.com/codeready-toolchain/host-operator/pull/946/commits/42d57507c680fd47418042c0f7dc4be07b7d19c2